rest/config.go

```diff
@@ -284,6 +287,38 @@ type Http2Config struct {
 	Enabled *bool `json:"enabled,omitempty"` // Whether HTTP2 support is enabled
 }
 
+//TODO: Add support for TLS 1.3 when we switch to Go 1.13
+
+var tlsVersionNames = []struct {
+	versionName  string
+	versionConst uint16
+}{
+	{
+		versionName:  "tlsv1",
+		versionConst: tls.VersionTLS10,
+	},
+	{
+		versionName:  "tlsv1.1",
+		versionConst: tls.VersionTLS11,
+	},
+	{
+		versionName:  "tlsv1.2",
+		versionConst: tls.VersionTLS12,
+	},
+}
+
+func GetTLSVersionFromString(stringV *string) *uint16 {
```

Comment:
Probably makes sense not to return a pointer here. We immediately dereference wherever we call this func, which has the possibility of panicing.

We'll always have a default value to return, and uint16 is small enough for the pass-by-value overhead not to matter.